cypress/support/tourNames.json

```diff
@@ -0,0 +1,9 @@
+{
+        "polishName":"Szczawnica test", 
```

Comment:
lepiejpisaczespacjamianiesadzisz?
czyli np: 
"polishName" : "Szczawnica test",

Reply:
masz racje tak jest lepiej nastepny commit ma to juz zrobione